examples/1D - Optimal Transport.ipynb

```diff
@@ -0,0 +1,5883 @@
+{
```

Comment:
I like using notebooks, because they are quick to make and easy to follow, specially if you are trying quickly understand how to use a package (it really helps me when repos have notebooks which I can just go to and follow along). I do agree that they are not well integrated with the documentation, but my suggestion would be to create examples in the documentation as you pointed out, AND leave a set o notebooks in the repo, so people can just go through them.
If you thinks this could get too messy, than I can just create another repository with Optimal Transport usage examples, and we can link to it in the README. I'm also gonna start a repo for generating OT plots.

Reply:
I would very much prefer a Literate example. They are Julia scripts with comments that can be exported to notebooks automatically (which can be included in the gh-pages branch) and included in the documentation, as the examples above demonstrate. And the additional advantage is that they are just regular Julia scripts so one can also just run them in the REPL without setting up a Jupyter environment first.

Reply:
I updated the outdated example in https://github.com/zsteve/OptimalTransport.jl/pull/47 and moved it to the suggested setup. The example can be viewed here: https://zsteve.github.io/OptimalTransport.jl/previews/PR47/examples/basic/ The Jupyter notebook can be viewed here: https://nbviewer.jupyter.org/github/zsteve/OptimalTransport.jl/blob/gh-pages/previews/PR47/examples/basic.ipynb

Reply:
This shows once more that there is really no need for adding notebooks to the repo even if you want to provide Jupyter notebooks for users.

Regarding plots, I think it might be best to just provide simple recipes with RecipesBase in OptimalTransport.jl.